css/style.css

```diff
@@ -303,14 +311,15 @@ h4 {
   z-index: -1;
 }
 
-.bookmark img{
+.bookmark img {
   width: 40px;
   margin-left: 20px;
  }
 
-.bookmark:hover{
+.bookmark:hover {
   -webkit-filter: drop-shadow(0 2px 6px hsla(0, 0%, 0%, 0.75));
 }
+}
```

Comment:
found this too.. 

![image](https://cloud.githubusercontent.com/assets/9439053/12629428/8b23c7d4-c50d-11e5-839e-02a9b3e23e6e.png)